server/routes/initialAppointment/handlers/initialMeetingTime.ts

```diff
@@ -21,7 +23,11 @@ export default class InitialMeetingTimeRoutes {
       appointmentTimeType,
       releaseIsOnBankHolidayOrWeekend: licence.isEligibleForEarlyRelease,
       skipUrl: this.getNextPage(licence.id.toString(), licence.typeCode, req),
-      canSkip: this.userType === UserType.PROBATION && !licence.appointmentTimeType,
+      canSkip:
+        this.userType === UserType.PROBATION &&
+        !licence.appointmentTimeType &&
+        licence.kind !== LicenceKind.VARIATION &&
+        !(config.hardStopEnabled && licence.isInHardStopPeriod),
```

Comment:
are people going to be able to access this screen when a licence is being varied when a value would not be set?
 
I think this could be simplified solely to `!licence.appointmentTimeType` - we discussed this whilst you were off and UX were happy with that.

Reply:
For completeness, access to the `initialMeetingTime` check is now controlled more tightly to cover user type, licence kind, and hard-stop status. This means we no longer need these additional checks here, so the `canSkip` check can be reduced to just `!licence.appointmentTimeType`